src/main/java/subway/path/domain/Path.java

```diff
@@ -0,0 +1,75 @@
+package subway.path.domain;
+
+import java.util.List;
+
+import org.jgrapht.GraphPath;
+import subway.path.exceptions.InvalidPathException;
+import subway.station.domain.Station;
+
+public class Path {
+    private final int BASE_FARE = 1_250;
+    private final int NON_DISCOUNTABLE_BASE_FARE = 350;
+    private final int FARE_PER_DISTANCE = 100;
+    private final int MIDDLE_DISTANCE_LIMIT = 10;
+    private final int FAR_DISTANCE_LIMIT = 50;
+    private final double FARE_INCREASE_MIDDLE_DISTANCE = 5.0d;
+    private final double FARE_INCREASE_FAR_DISTANCE = 8.0d;
```

Comment:
`5.0`과 같이 실수형으로 표기한 시점에서 이미 `double` 확정이라 `d` suffix는 꼭 붙이지 않으셔도 됩니다.
가장 짧은 표기형은 아래처럼 되겠네요 (둘 모두 가능한 표현)
```suggestion
    private final double FARE_INCREASE_MIDDLE_DISTANCE = 5d;
    private final double FARE_INCREASE_FAR_DISTANCE = 8.;
```